metadatastore/commands.py

```diff
@@ -126,8 +126,8 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     beamline_id : str
         Beamline String identifier. Not unique, just an indicator of
         beamline code for multiple beamline systems
-    beamline_config : metadatastore.odm_temples.BeamlineConfig
-        Foreign key to beamline config corresponding to a given run
+    beamline_config : str
+        uid of beamline config corresponding to a given run
```

Comment:
However, these (and similar params in other insert functions) should be renamed to `beamline_config_uid`